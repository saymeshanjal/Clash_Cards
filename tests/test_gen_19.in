90
6 248 3
2 185 3
10 238 3
12 39 3
16 291 3
16 31 4
2 304 4
6 53 5
4 64 2
10 294 2
10 384 2
6 217 2
12 341 5
14 3 5
2 18 3
8 337 4
4 153 5
6 340 3
12 189 4
10 9 4
14 85 3
2 144 2
8 221 1
12 11 4
14 124 2
6 375 1
12 393 5
4 291 2
4 128 1
6 284 3
4 146 3
2 266 4
4 78 1
16 171 4
2 233 5
8 59 1
16 128 1
12 297 2
16 171 2
16 213 2
2 270 5
14 326 4
12 199 4
4 182 5
2 230 5
10 9 4
16 161 2
8 390 2
6 334 3
4 85 3
12 53 3
10 275 4
12 96 4
10 89 1
14 266 2
4 254 3
16 29 1
12 32 2
12 10 3
2 187 3
2 353 1
14 205 1
8 10 5
16 198 4
4 92 2
6 373 3
12 156 2
2 65 1
6 351 1
6 335 4
16 280 4
4 310 4
16 1 1
12 0 4
10 42 1
14 356 1
12 105 2
6 250 4
14 284 5
12 239 1
16 150 3
6 24 5
6 135 2
10 132 2
10 200 4
2 14 3
4 324 5
14 262 5
10 260 2
6 281 1
