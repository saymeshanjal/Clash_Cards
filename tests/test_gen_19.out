6 14 0
0 2 0
110 142 0
309 615 0
6307 6563 0
343 10545 1982
0 2 0
1 19 0
1 2 3
76 77 99
76 77 99
5 6 9
219 705 0
0 3432 0
0 2 0
8 42 20
0 6 0
6 14 0
208 441 275
0 235 17
1652 1780 0
0 1 1
35 35 0
4 863 57
1078 1079 1275
10 10 0
219 705 0
1 2 3
3 3 0
6 14 0
1 5 0
0 2 0
3 3 0
3809 5406 3655
0 2 0
35 35 0
6435 6435 0
286 287 351
4081 4082 4707
4081 4082 4707
0 2 0
910 1520 1002
208 441 275
0 6 0
0 2 0
0 235 17
4081 4082 4707
20 21 29
6 14 0
1 5 0
430 494 0
44 133 75
208 441 275
126 126 0
1078 1079 1275
1 5 0
6035 6835 0
86 134 704
57 867 0
0 2 0
1 1 0
1716 1716 0
1 69 0
3809 5406 3655
1 2 3
6 14 0
286 287 351
1 1 0
10 10 0
1 14 5
3809 5406 3655
0 5 1
6435 6435 0
0 924 0
126 126 0
1716 1716 0
286 287 351
1 14 5
987 2445 0
462 462 0
6307 6563 0
1 19 0
5 6 9
76 77 99
44 133 75
0 2 0
0 6 0
987 2445 0
76 77 99
10 10 0
